build.fsx

```diff
@@ -364,7 +364,7 @@ Target "MergeDotnetCoreIntoNuget" (fun _ ->
 
     let runTool = runCmdIn "tools" dotnetExePath
 
-    runTool """mergenupkg --source "%s" --other "%s" --framework netstandard1.6 """ nupkg netcoreNupkg
+    runTool """0x53A-mergenupkg --source "%s" --other "%s" --framework netstandard1.6 """ nupkg netcoreNupkg
```

Comment:
I think this fixes https://github.com/fsprojects/Paket/issues/2645. This is bad. We didn't even realize that we still depend on SDK1.0 to be installed. Thanks :)

Reply:
You are welcome ;)